docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,235 @@
+
+## What data do I need?
+
+What to consider when chosing data:
+
+* Sensor
+    * Optical
+    * Multispectal
+    * Hyperspectral
+    * RADAR (Radio Detection and Ranging), SAR (Synthetic Aperture Radar)
+    * LiDAR (Light Detection and Ranging)
+* Resolution
+    * Temporal: when and how often a certain area is visited
+    * Spatial: the area on the ground that each pixel covers
+    * Spectral: the area of the electromagnetic spectrum that is observed and spectral width of each band provided
+        * Depending on the wavelengths observed,  clouds and atmospheric artifacts may result in data gaps
+    * Radiometric: how many values are possible for each pixel (bit-depth)
+* Costs:
+    * Some EO data is freely available as open data
+    * Some commercial datasets might be possible to get for free/less for research
+* Preprocessing level
+    * Raw data
+    * Analysis ready data 
+    * Mosaics
+* User experience and knowledge
+    * RADAR/LiDAR require solid background knowledge for processing and interpretation
+    * Optical data is more easily interpreted and processed (and more pre-processed data is available)
+
+### Some widely used EO datasets
+
+|Name|Max resolution, m|Revisit time, days|Years of operation|Open data|
+|--------------|------------|-----|---------|---------|
+|**Optical, multispectral**|
+|**[ESA, Sentinel-2](https://sentinel.esa.int/web/sentinel/missions/sentinel-2)**|10-60|5|2015->|Yes|
+|**[NASA, Landsat](https://landsat.gsfc.nasa.gov/)**|15-120|8|1972->|Yes|
+|[ESA, Proba-V](https://earth.esa.int/eogateway/catalog/proba-v-1km-333m-and-100m-products)|100-1000|1-2|2013->|Yes|
+|[Airbus, Spot](https://www.intelligence-airbusds.com/imagery/constellation/spot/)|1.5|-|1986->|No|
+|[Planet, several satellites ](https://www.planet.com/products/planet-imagery/)|0.5-5|-|2009-> |No|
+|[DigitalGlobe, WorldView](http://worldview3.digitalglobe.com/)|0.3-30|-|1997->|No|
+|[Airbus, Pleiades](https://pleiades.cnes.fr/en/PLEIADES/index.htm)|0.3-0.5|-|2012->|No|
+||||||
+|**Hyperspectral**|||||
+|**[NASA, MODIS](https://modis.gsfc.nasa.gov/)**|250-100|1-2|1999->|Yes|
+|[NASA, EO-1](https://www.usgs.gov/centers/eros/science/earth-observing-1-eo-1)|10-30|-|2000-2017|Yes|
+||||||
+|**Radar, SAR**|||||
+|**[ESA, Sentinel 1](https://sentinel.esa.int/web/sentinel/missions/sentinel-1)**|5|6|2014->|Yes|
+|[ESA, Radarsat](https://www.asc-csa.gc.ca/eng/satellites/radarsat/)|1-100|24|1995->|Yes|
+|[TanDEM-X](https://www.dlr.de/content/en/missions/tandem-x.html)/[TerraSAR-X](https://www.dlr.de/content/en/missions/terrasar-x.html)|0.25-40|-|2010->|No|
+|[ICEYE](https://www.iceye.com/)|0.5-2.5|1|2018->|No|
+||||||
+|**LiDAR**|Footprint size||||
+|[NASA, ICESat2](https://icesat-2.gsfc.nasa.gov/)|13|91|2019->|Yes|
+|[NADA, GEDI](https://gedi.umd.edu/)|25|-|2018->|Yes|
```

Comment:
Should we add MML lidar here? Or maybe it was already mentioned elsewhere

Reply:
I my mind I have put the line on satellite data here, so I would not may-be put it here. Put if you put NLS lidar, then NLS orthoimages should also be added.

Reply:
Jup, I wuold also stick to satellites, Earth Observation as in observing whole earth, in a way. Will add a note in beginning